vmrss

```diff
@@ -27,7 +27,7 @@ while [ ${#arr[@]} -gt 0 ]; do
       | grep $GREP_OPTS -o '[0-9]\+' \
       | awk '{print $1/1024}')
     #Add decimals to total
-    total = $(echo $mem+$total | bc)
+    total=$(echo $mem+$total | bc)
```

Comment:
we don't need to fork to accomplish this
```suggestion
    ((total+=mem))
```

Reply:
Thanks for the advice!
I think it works for bash 5+?
I tested it on bash 4.4 and got this error
```
./vmrss.sh: line 30: ((: total+=43863.9: syntax error: invalid arithmetic operator (error token is ".9")
```

Reply:
oh you're right it doesn't work with decimals rip